samcli/lib/utils/managed_cloudformation_stack.py

```diff
@@ -60,11 +60,13 @@ def manage_stack(
     """
     try:
         if profile:
-            session = boto3.Session(profile_name=profile, region_name=region if region else None)  # type: ignore
+            session = boto3.Session(profile_name=profile, region_name=region if region else None)
             cloudformation_client = session.client("cloudformation")
         else:
+            # botocore-stubs does not allow Optional region_name, will be fixed soon
             cloudformation_client = boto3.client(
-                "cloudformation", config=Config(region_name=region if region else None)
+                "cloudformation",
+                config=Config(region_name=region if region else None),  # type: ignore
             )
```

Comment:
`Config` works correctly with `region_name=None`. I have added a fix to `botocore-stubs`.

Reply:
New version released, `type: ignore` removed.